### ISSUE TITLE
Missing alt text warning is missing

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#1056]
I think we've dropped the missing alt text warning that we used to show in the left sidebar

<img width="1525" height="663" alt="Image" src="https://github.com/user-attachments/assets/cdeb0d5e-d962-46ed-b341-d1fe42945efa" />
